app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -221,6 +269,15 @@ class ParkingViewModel(
     }
   }
 
+  /**
```

Comment:
Basically everywhere so imma stop here, you get the point

Reply:
Yes I do think that the rebasing was a bit messy. I will take a look at that

Reply:
No problem, thanks !